graphql/coding/userBadges/fetchUserBadges.ts

```diff
@@ -17,14 +17,18 @@ export const FETCH_CODING_BADGES = gql`
   }
 `;
 
-type Response = {
+export type Response = {
```

Comment:
specific need to export this type globally? I could see this causing some issues because it's named pretty generically